src/UI/src/components/MessageListTable/MessageListTable.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { messagePath } from "../../paths";
+
+const useStyles = makeStyles({
+  rowStyle: {
+    cursor: "pointer",
+    textDecoration: "none",
+  },
+  loadMoreButtonPosition: {
+    textAlign: "center",
+    marginTop: "40px",
+    marginBottom: "40px",
+  },
+});
+
+interface MessageList {
+  readonly messages: {
+    readonly edges: ReadonlyArray<{
+      readonly node: {
+        readonly id: string;
+        readonly title: string;
+        readonly to: ReadonlyArray<string | null> | null;
+        readonly receivedAt: string;
+        readonly type: string;
+        readonly provider: string;
+      } | null;
+    }> | null;
+  } | null;
+  hasNext: boolean;
+  loadNext(count: number): void;
+}
+
+const MessageListTable: React.FC<MessageList> = ({
+  messages,
+  hasNext,
+  loadNext,
+}) => {
+  const classes = useStyles();
+  const navigate = useNavigate();
+
+  const getDateTime = (date: any) => new Date(date).toLocaleString() ?? "";
+
+  const getShortTitle = (title: string) =>
+    title.length > 50 ? title.substring(0, 50) + "..." : title;
+
+  return (
+    <>
```

Comment:
remove

Reply:
not possible, because there is a div under the table